### PR TITLE
34 replacement file

### LINKED ISSUE #34: Generate replacement file
Just take all deprecated entries that have an id.

### PR DESCRIPTION
**Pull request by fenekku**
- closes #34

### CODE REVIEW

**Comment by fenekku** [comment]
Simple follow-up to previous PR.

**Review by mattbaumann1 — APPROVED** [review]
Seems pretty similar to the last one.

**Review by sharpattack — APPROVED** [review]

**Review by Meowcenary — APPROVED** [review]
Looks good to me 👍

File: galter_subjects_utils/contrib/lcsh/converter.py [hunk 1]
```diff
@@ -106,6 +109,23 @@ def is_deprecated(topic):
     return bool(find_deprecation_node(topic, node_main))
 
 
+def happened_since(topic, since=None):
+    """Filter for topic having happened after since.
+
+    :param topic: skos topic
+    :type since: dict
```
Comment (Meowcenary) [inline galter_subjects_utils/contrib/lcsh/converter.py:116]: change to ":type topic: dict"

**Review by Meowcenary — COMMENTED** [review]
Caught one thing on a second look

File: galter_subjects_utils/contrib/lcsh/converter.py (hunk 1 above)
Comment (fenekku) [reply galter_subjects_utils/contrib/lcsh/converter.py]: I just rebased it, so those should go away with the fix from the previous PR :wink:

**Pull request merged**